src/pages/NotificationsPage.vue

```diff
@@ -145,14 +145,24 @@ import commonMixin from '../util/common-mixin'
 
 export default {
   mixins: [commonMixin],
-
+  computed: {
+    showNotificationTab() {
```

Comment:
Why are there 2 showNotification getters with different signatures? Are both of them used?

Reply:
The one on line 149 is the getter. In line 157 the page is watching for changes in the value that showNotificationTab is getting.